components/brave_wallet_ui/panel/container.tsx

```diff
@@ -735,20 +712,12 @@ function Container() {
     )
   }
 
-  if (isPanelV2FeatureEnabled) {
-    return (
-      <PanelWrapper
-        width={390}
-        height={650}
-      >
-        <PageContainer />
-      </PanelWrapper>
-    )
-  }
-
   return (
-    <PanelWrapper isLonger={false}>
-      <ConnectedPanel navAction={navigateTo} />
+    <PanelWrapper
+      width={390}
+      height={650}
+    >
+    <PageContainer />
```

Comment:
`ConnectedPanel` component and related stuff also should be removed I think

Reply:
@Douglashdaniel Should I remove all these?
```
components/brave_wallet_ui/components/extension/connected-panel/index.tsx:export const ConnectedPanel = (props: Props) => {
components/brave_wallet_ui/components/extension/connected-panel/index.tsx:export default ConnectedPanel
components/brave_wallet_ui/stories/wallet-extension-panels.tsx:import { ConnectedPanel } from '../components/extension/connected-panel/index'
components/brave_wallet_ui/stories/wallet-extension-panels.tsx:export const _ConnectedPanel = (args: { locked: boolean }) => {
components/brave_wallet_ui/stories/wallet-extension-panels.tsx:              <ConnectedPanel navAction={navigateTo} />
components/brave_wallet_ui/stories/wallet-extension-panels.tsx:_ConnectedPanel.args = {
components/brave_wallet_ui/stories/wallet-extension-panels.tsx:_ConnectedPanel.story = {
```

Reply:
@yrliou and @supermassive I can handle this separately! Am planning on removing all `unused` components for the Wallet `Front-end`  in a PR now that `Wallet V2` is in release.

Reply:
@supermassive okay if I leave them as is for this PR?

Reply:
Ok to do that separately!